### ISSUE TITLE
Navigation Elements Not Displaying on Small Screens

### ISSUE THREAD

**Issue by RohithChitrada** [ayush-that/FinVeda#462]
### What happened?

When the screen width is below 430px, the toggle button for the navigation menu is displayed correctly, but the navigation elements (links) are not visible.

![Screenshot 2024-07-14 105947](https://github.com/user-attachments/assets/7d9811fc-8bc3-45f0-b002-bdf3937c624e)

### How can we reproduce this bug?

1. Open the application in a web browser.
2. Resize the browser window to a width below 430px.
3. Observe the navigation bar.

### Urgency (Optional)

High priority

**Comment by RohithChitrada** [comment]
Hi @deepeshmlgupta,

Issue Description:

1. Package.json Not Found: When I attempt to run the project, I receive an error stating that the package.json file is not found.
2. Running Another Folder: I tried running a different Finveda folder, and while it started, the output only displays the Vite and React logos, without the expected functionality.

Steps Taken:
Cloned the repository from GitHub.
Navigated to the project directory.
Ran npm install to install dependencies.
Executed npm run dev to start the development server.

Additional Information:
I confirmed that package.json exists in the original project folder.
In the second folder, the output is not what I anticipated; it seems incomplete.

I would appreciate any guidance on resolving these issues. Thank you for your help!
